src/m365/viva/commands/engage/engage-role-member-list.spec.ts

```diff
@@ -0,0 +1,209 @@
+        return {
+          "value": [
+            {
+              "id": "1f5595b2-aa07-445d-9801-a45ea18160b2",
+              "createdDateTime": "2024-05-22T15:43:08.368Z",
+              "userId": "1f5595b2-aa07-445d-9801-a45ea18160b2"
+            },
+            {
+              "id": "717f1683-00fa-488c-b68d-5d0051f6bcfa",
+              "createdDateTime": "2025-07-16T02:51:22.602Z",
+              "userId": "717f1683-00fa-488c-b68d-5d0051f6bcfa"
+            }
+          ]
+        };
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: { verbose: true, roleId: roleId }
+    });
+
+    assert(
+      loggerLogSpy.calledOnceWith([
+        {
+          "id": "1f5595b2-aa07-445d-9801-a45ea18160b2",
+          "createdDateTime": "2024-05-22T15:43:08.368Z",
+          "userId": "1f5595b2-aa07-445d-9801-a45ea18160b2"
+        },
+        {
+          "id": "717f1683-00fa-488c-b68d-5d0051f6bcfa",
+          "createdDateTime": "2025-07-16T02:51:22.602Z",
+          "userId": "717f1683-00fa-488c-b68d-5d0051f6bcfa"
+        }
+      ])
+    );
+  });
+
+  it(`should get a list of users assigned to a Viva Engage role specified by names`, async () => {
+    sinon.stub(vivaEngageRole, 'getRoleIdByName').resolves(roleId);
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/beta/employeeExperience/roles/${roleId}/members`) {
+        return {
+          "value": [
+            {
+              "id": "1f5595b2-aa07-445d-9801-a45ea18160b2",
+              "createdDateTime": "2024-05-22T15:43:08.368Z",
+              "userId": "1f5595b2-aa07-445d-9801-a45ea18160b2"
+            },
+            {
+              "id": "717f1683-00fa-488c-b68d-5d0051f6bcfa",
+              "createdDateTime": "2025-07-16T02:51:22.602Z",
+              "userId": "717f1683-00fa-488c-b68d-5d0051f6bcfa"
+            }
+          ]
+        };
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: { verbose: true, roleName: roleName }
+    });
+
+    assert(
+      loggerLogSpy.calledOnceWith([
+        {
+          "id": "1f5595b2-aa07-445d-9801-a45ea18160b2",
+          "createdDateTime": "2024-05-22T15:43:08.368Z",
+          "userId": "1f5595b2-aa07-445d-9801-a45ea18160b2"
+        },
+        {
+          "id": "717f1683-00fa-488c-b68d-5d0051f6bcfa",
+          "createdDateTime": "2025-07-16T02:51:22.602Z",
+          "userId": "717f1683-00fa-488c-b68d-5d0051f6bcfa"
+        }
+      ])
+    );
+  });
+
+  it('handles error when retrieving Viva Engage roles failed', async () => {
+    sinon.stub(request, 'get').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/beta/employeeExperience/roles/${roleId}/members`) {
+        throw { error: { message: 'An error has occurred' } };
+      }
+      throw `Invalid request`;
+    });
```

Comment:
In this case, we just want to test if the command correctly displays the error message created by the API.
We don't really care whether the right API request is created, this is already validated by the other tests. Therefore I suggest that we remove the URL check.